doc/styles/Typography.md

```diff
@@ -0,0 +1,78 @@
+# Typography
```

Comment:
Should this file be named Typography or TextBlock?

Reply:
@Xiaoy312 / @agneszitte 
 
 The styles are TextBlock styles so it should be in a TextBlock.md, as for the resources, these are more general Typography resources, not TextBlock related so maybe we should split these into a TextBlock.md and Typography.md?

Reply:
this file (and all reference to here) should be renamed to `TextBlock,md`

`typography.md` should be reserved for resources from: src\library\Uno.Material\Styles\Application\v2\Typography.xaml
although currently there is no plan for adding this

Reply:
@Xiaoy312 should I just remove the Lightweight Styling section from the new TextBlock.md then?